### ISSUE TITLE
 tested Prediff 

### ISSUE THREAD

**Issue by zhouy02** [gaozhihan/PreDiff#9]
I have tested Prediff based on your code and found  differences in the results compared to the paper. I wonder if you could provide some suggestions.
<img width="225" alt="test" src="https://github.com/gaozhihan/PreDiff/assets/104981581/4489cb10-d8d0-42b4-9621-8c58befb4530">

**Comment by zhouy02** [comment]
Thank you very much for your response, but I still have some questions and need your assistance. 1. I chose 'eval_example_only: false', 'eval_unaligned: true', 'eval_aligned: false'. Therefore, I conducted testing on the entire test set based on the pre-trained model you provided. However, the obtained results show a generally low CSI value around 0.1-0.25, and there is a significant difference in bias. Is this possibly related to the following part of the code? 2. Approximately, how long does it take to perform testing using the 'ddpm' code in the script?
![image](https://github.com/gaozhihan/PreDiff/assets/104981581/34f42c3a-84f0-4ec5-b1a0-1c5ca8b4a18f)

**Comment by forevergj** [comment]
> Thank you for your question. You may want to refer to Section D.2 of the Appendix in our [paper](https://openreview.net/pdf?id=Gh67ZZ6zkS). In brief, simply average multiple samples results in better CSI scores.

请问论文中计算cis指标的测试集是如何划分的？
